sklearn/neural_network/tests/test_mlp.py

```diff
@@ -556,3 +561,40 @@ def test_adaptive_learning_rate():
     clf.fit(X, y)
     assert_greater(clf.max_iter, clf.n_iter_)
     assert_greater(1e-6, clf._optimizer.learning_rate)
+
+
+def test_warm_class():
+    X = X_iris[:150]
+    y = y_iris[:150]
+
+    y_2classes = np.array([0]*75 + [1]*75)
+    y_3classes = np.array([0]*50 + [1]*50 + [2]*50)
+    y_4classes = np.array([0]*37 + [1]*37 + [2]*38 + [3]*38)
+    y_5classes = np.array([0]*30 + [1]*30 + [2]*30 + [3]*30 + [4]*30)
+
+    with ignore_warnings(category=Warning):
+        # failed in converting 7th argument `g' of _lbfgsb.setulb to
```

Comment:
Yes, these are the errors that occurs originally and are replaced with a standard error message. I will remove these comments.